src/main/java/duke/task/FileClass.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class FileClass {
```

Comment:
This also does not seem to be implemented with your `main` for class `Duke`. Perhaps you might want to include and use this class there?